src/main/kotlin/test/ktortemplate/core/utils/healthcheck/HealthCheck.kt

```diff
@@ -8,11 +9,17 @@ internal class HealthCheckInjector : KoinComponent {
     val dbc: DatabaseConnection by inject()
 }
 
-fun Health.Configuration.liveness() {
-    liveCheck("alive") { true }
-}
+fun Health.Configuration.liveness() = liveCheck("alive") { true }
```

Comment:
Shouldn't the liveness check also use the database? I.E. this is the check that is supposed to be used when the application is up and running.

Reply:
I'm basing this on the definition of liveness and readyness probes here:
https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes

Reply:
not sure about that. Based on the same definition, liveness should check if the system is "dead", so it shouldn't go much deep. On the other hand, readiness should check if the system is ready to serve traffic, so it should go deep and check if everything is in place.
Besides that, what is the point of having the same check in both probes? If both have the same checks, we can use just one endpoint for both liveness and readiness

Reply:
Yeah, I get your point but I got a bit of mixed feelings for that in the sense that during operation (where the liveness will be called repeatedly), I want things to warn me when the database goes "down" - i.e. takes so long to respond to the simplest possible query that it is essentially dead.
By not doing that check in the liveness probe, you risk having a liveness probe that says that everything is Okay even when there is no database connection.

Reply:
isn't readiness prob configurable, just like liveness? Why switch concepts when you can just set a lower interval between readiness endoint calls?